### DISCUSSION TITLE
Advanced Prices has a new element

### DISCUSSION THREAD

**Discussion by madpilot** [amberelectric/public-api#218] [Announcements]
Now that we've added the Amber forecast price line to the graphs in Smartshift, we've added a new corresponding element called `predicted` to the advance price attribute.

You can use this if you need a single number for forecasts in your own automations.

The Home assistant plugin is coming along nicely. I hope to have it submitted some time this week.

**Comment by hags86** [comment]
Hey Madpilot
Having given up waiting for the Fronius/BYD integration to hit the streets I've just started researching Home Assistant route and saw this post.  What are you cooking up with HA in terms of scope/features?

**Reply by madpilot** [reply]
The Smartshift API is still a while off, but FroniusBYD It should be in the next week or. I can get you on the test beta today if you like? Email info@amber.com.au Attn: Myles and give me your serial number off you inverter.

**Comment by hags86** [comment]
OK. I'll sign up with you guys today and then send the email once I have my Amber account set up.

**Comment by altonius** [comment]
@madpilot how's the home assistant plugin going?

**Reply by madpilot** [reply]
Slowly. I haven't really had time to work on it. I just need to write some tests.

**Reply by madpilot** [reply]
PR is here: https://github.com/home-assistant/core/pull/124008

Now we play the "waiting for merge" game.
